icu4c/source/common/umutex.h

```diff
@@ -336,13 +190,6 @@ U_NAMESPACE_END
 #error U_USER_MUTEX_H not supported
```

Comment:
Don't need the check down here any more since you moved it up with the other check at the top of the file

Reply:
Done